### DISCUSSION TITLE
Is it possible to scale the gui?

### DISCUSSION THREAD

**Discussion by Awesomecow67** [vinegarhq/sober#261] [Q&A] [answered]
The gui on all the roblox games I play is very small. Is there a setting somewhere I can change to fix it?
![image](https://github.com/user-attachments/assets/477456ca-142e-4c1d-9259-5ee4c78468f4)
![image](https://github.com/user-attachments/assets/50f509d4-5e9c-4005-acb0-cef35df5816f)

**Comment by olifurz** [comment]
Not that I know of apart from reducing your screen resolution.

**Comment by Bugsbane** [comment]
Similarly, I find that a lot of places where you have to enter text, the font goes tiny. Not sure why:
![image](https://github.com/user-attachments/assets/13887a54-5448-4644-966b-251234434774)
Image is from Sol's RNG

**Comment by davidcollini** [comment]
Been having the same issue, I haven’t gotten around to changing the screen resolution, but it would be great to see a setting to scale the Ui built into Sober

**Comment by sewnie** [comment, marked as answer]
https://github.com/vinegarhq/sober/issues/636
